build.psm1

```diff
@@ -2282,7 +2282,6 @@ function Start-CrossGen {
         "Microsoft.PowerShell.Security.dll",
         "Microsoft.PowerShell.CoreCLR.Eventing.dll",
         "Microsoft.PowerShell.ConsoleHost.dll",
-        "Microsoft.PowerShell.PSReadLine.dll",
```

Comment:
I'd imagine this will hurt startup, but maybe it can't be cross-gen'd.

Reply:
I think you should also remove the Windows specific dll `System.Runtime.InteropServices.RuntimeInformation.dll` from the nuget package - PowerShell Core ships the correct version of this file and this copy is just wasting disk space and I suppose could cause problems in the future.

Reply:
Had to remove from crossgen list as the crossgen happens before psmodulerestore.  We can reorder it if startup time is an issue.  The code to not include `System.Runtime.InteropServices.RuntimeInformation.dll` is already there at line 2395